README.md

```diff
@@ -9,7 +9,7 @@ Looking for V1 of the .NET Core tooling?
 
 If you are looking for the v2.0 release of the .NET Core tools (CLI, MSBuild and the new csproj), head over to https://dot.net/core and download!
 
-> **Note:** the dev/repo-refactoring branch of the .NET Core SDK repo is based on an upcoming update of the SDK and is considered pre-release. For production-level usage, please use the
+> **Note:** the master branch of the .NET Core SDK repo is based on an upcoming update of the SDK and is considered pre-release. For production-level usage, please use the
 > released version of the tools available at https://dot.net/core
```

Comment:
Should this point to https://get.dot.net instead?

Reply:
This links takes people to a little tutorial.